### DISCUSSION TITLE
Outdated documentation

### DISCUSSION THREAD

**Discussion by kpodsiad** [scala/scala3#16090] [General Discussion]
I believe that https://github.com/lampepfl/dotty/blob/fc8b56ef412538fbccb726d746e82bb122820283/docs/_docs/reference/language-versions/binary-compatibility.md is pretty much outdated after [announcement of LTS version ](https://www.scala-lang.org/blog/2022/08/17/long-term-compatibility-plans.html). 

After closer look, all of [language version sections](https://github.com/lampepfl/dotty/tree/fc8b56ef412538fbccb726d746e82bb122820283/docs/_docs/reference/language-versions) needs minor updates to stay up to date.

**Comment by bishabosha** [comment]
the idea of compatibility has not changed, LTS just means maintaining more than one stable line of releases. The link you gave is outdated since we removed mention of `-scala-output-version` already

**Reply by kpodsiad** [reply]
You're right, the changes were made on github on main branch. However, they're still present on the https://docs.scala-lang.org/scala3/reference/language-versions/binary-compatibility.html and I went to github by clicking edit this page github button.

There's scala version set to `3.1.3` but there's no dropdown to pick newer version

**Reply by bishabosha** [reply]
Oh that's unfortunate, it should have been updated when 3.2.0 was announced.

@pikinier20 do you know what's happening here?
